sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/impl/rule/BeamAggregateProjectMergeRuleTest.java

```diff
@@ -106,44 +106,6 @@ public void testBeamAggregateProjectMergeRule_withProjectTable_withPredicate() {
         ioSourceRel.getRowType().getFieldNames(), containsInAnyOrder("name", "id", "unused1"));
   }
 
-  @Test
```

Comment:
Can we keep the test with `@Ignore("BeamAggregateProjectMergeRule disabled due to <calcite bug link>")` ?